### PR TITLE
[SMN] Correct Fenrir animations & Blood Pact abilities

### PR DESCRIPTION
**Pull request by N3ckB3ard**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Updates all of Fenrir's animations to reflect new pet_skills sql table
Compared all Blood pact msg's to retail captures
Added new msg enums to reflect retail captures
Removes Fenrir's animation shifting from core
Added SQL updates for future Fenrir Blood Pact abilities

## Steps to test these changes

Use Fenrir's BP Ward to see proper MP consumption and messages displayed
Use Fenrir's BP Rage to see proper MP consumption and messages displayed
Tested animations of future Blood Pact abilities

### CODE REVIEW

File: scripts/actions/abilities/pets/lunar_roar.lua [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+abilityObject.onPetAbility = function(target, pet, petskill, summoner, action)
+    xi.job_utils.summoner.onUseBloodPact(target, petskill, summoner, action)
+
+    local effect = target:dispelStatusEffect()
+    if effect == xi.effect.NONE then
+        petskill:setMsg(xi.msg.basic.JA_NO_EFFECT_2)
+    else
+        target:dispelStatusEffect()
+        target:dispelStatusEffect()
+        petskill:setMsg(xi.msg.basic.NONE)
```
Comment (WinterSolstice8) [inline scripts/actions/abilities/pets/lunar_roar.lua:18]: Wouldn't this be dispelling 3 status effects (wiki says two effects) and then not reporting a dispel?

Comment (N3ckB3ard) [reply scripts/actions/abilities/pets/lunar_roar.lua:18]: Retail doesn't report when a effect is dispelled based on captures but I see what you mean. I can't remember if I intended to remove the additional dispel or left it from testing. I will have to report back on this

Comment (N3ckB3ard) [reply scripts/actions/abilities/pets/lunar_roar.lua:18]: I corrected the issue of the effects. This is a reference to retail with no display message of removal 
image

When nothing can be removed, the client is sent "no effect" message
image

Comment (WinterSolstice8) [reply scripts/actions/abilities/pets/lunar_roar.lua:18]: Interesting. The only difference is the "reaction" other than the message but I don't know if that actually produces a different message to the client. Thanks for the caps.

**Review by WinterSolstice8 — APPROVED** [review]

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**